classes/tdx-encrypted.bbclass

```diff
@@ -41,6 +41,13 @@ TDX_ENC_STORAGE_RESERVE ?= "0"
 # Defines where the encrypted storage will be mounted
 TDX_ENC_STORAGE_MOUNTPOINT ?= "/run/encdata"
 
+# Extra arguments passed to "mkfs" when creating the filesystem on top
+# of the encrypted storage
+TDX_ENC_STORAGE_MKFS_ARGS ?= ""
+
+# Extra arguments passed to "mount" when mounting the encrypted storage
+TDX_ENC_STORAGE_MOUNT_ARGS ?= ""
```

Comment:
Nice addition! Maybe I am missing something, but did you added those variables to the documentation? If not, would you do it please?

Reply:
I added that generic message asking users to see the class file for the full list of variables. Before, there was already variables related to the data preservation that were not documented either. What should we do? Should we document all indeed?